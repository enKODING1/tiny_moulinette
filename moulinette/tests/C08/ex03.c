#include <stdio.h>
#include "ft_point.h"

int main(void)
{
    t_point point;

    point.x = 4;
    point.y = 2;
    printf("point.x: %d, point.y: %d\n", point.x, point.y);
    return 0;
}
