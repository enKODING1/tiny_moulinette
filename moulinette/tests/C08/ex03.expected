point.x: 4, point.y: 2
